client/state/guided-tours/selectors/index.js

```diff
@@ -189,7 +190,7 @@ export const getGuidedTourState = createSelector(
 		);
 
 		if ( ! tour ) {
-			return { ...tourState, ...emptyState };
```

Comment:
Is it possible for `getRawGuidedTourState( state );` to include data which is different from EMPTY_STATE? (Since we're spreading both together here)

Reply:
I don't think so. It will only contain any data if a tour is being started, resumed, or paused, which will only be true if there's a `tour` object, as far as I understand. Since this check ensures thare's no `tour`, there will also be no `tourState`.